app/Repositories/H5pLibrary/H5pLibraryRepository.php

```diff
@@ -0,0 +1,184 @@
+            $fields['indexed'] = false;
+            $fields['label'] = '';
+            $fields['description'] = '';
+
+            if ($array[$key] == 'text') {
+                $pathTypeCount = count($pathType);
+
+                if ($pathTypeCount > 2 && $pathType[$pathTypeCount - 3] == 'list' && $pathType[$pathTypeCount - 2] == 'group'){
+                    // unset($pathType[$pathTypeCount - 2]);
+                    // unset($pathName[$pathTypeCount - 2]);
+
+                    $pathName[$pathTypeCount - 2] = 'group.nested.array';
+
+                    if (
+                        isset($pathType[$pathTypeCount - 4])
+                        && $pathType[$pathTypeCount - 4] == 'group'
+                        && $pathName[$pathTypeCount - 4] == $pathName[$pathTypeCount - 3]
+                    ){
+                        unset($pathType[$pathTypeCount - 4]);
+                        unset($pathName[$pathTypeCount - 4]);
+                    }
+                }
+
+                if ($pathTypeCount > 1){
+                    if (isset($pathType[$pathTypeCount - 2]) && $pathType[$pathTypeCount - 2] == 'list'){
+                        unset($pathType[$pathTypeCount - 1]);
+                        unset($pathName[$pathTypeCount - 1]);
+
+                        if (isset($pathType[$pathTypeCount - 3])
+                            && $pathType[$pathTypeCount - 3] == 'group'
+                            && isset($pathType[$pathTypeCount - 4])
+                            && $pathType[$pathTypeCount - 4] == 'list'
+                        ) {
+                            unset($pathType[$pathTypeCount - 3]);
+                            unset($pathName[$pathTypeCount - 3]);
+                        }
+                    }
+                }
+
+                if ($pathTypeCount > 3){
+                    if (
+                        isset($pathType[$pathTypeCount - 4]) && $pathType[$pathTypeCount - 4] == 'list' &&
+                        isset($pathType[$pathTypeCount - 3]) && $pathType[$pathTypeCount - 3] == 'group' &&
+                        isset($pathType[$pathTypeCount - 2]) && $pathType[$pathTypeCount - 2] == 'group'
+                    ){
+                        unset($pathType[$pathTypeCount - 3]);
+                        unset($pathName[$pathTypeCount - 3]);
+                        unset($pathType[$pathTypeCount - 2]);
+                        unset($pathName[$pathTypeCount - 2]);
+                    }
+                }
+
+                $fields['path'] = implode('.', $pathName);
+            }
+
+            if ($nameLike) {
+                if (in_array(strtolower($array['name']), $nameLike, true)){
+                // if (Str::contains(strtolower($array['name']), $nameLike))
+                    $fields['library_id'] = $libraryId;
+                    $fields['name'] = $array['name'];
+                    $fields['type'] = $array['type'];
+
+                    if (isset($array['label']))
```

Comment:
Wrap your if condition body in brackets, even if it's in a single line.
There are many occurrences of this in the code.

Also, please correct position of opening and closing braces for all if conditions.

Reply:
Added brackets for single line if statements.